royalties-registry/migrations/5_royalties_provider_art_blocks.js

```diff
@@ -69,15 +69,12 @@ async function setArtBlocksProvider(deployer, network, royaltiesRegistry, settin
         return;
     }
 
-    await RoyaltiesProviderArtBlocks.deployed().catch(
-        async () => {
-            const contract = await deployer.deploy(RoyaltiesProviderArtBlocks, settings.artblocksAddress,  { gas: 1000000 });
-            console.log(`set artblocksAddress ${settings.artblocksAddress} for royaltiesProviderArtBlocks ${contract.address}`)
-            for (const token of settings.tokens){
-                await royaltiesRegistry.setProviderByToken(token, contract.address,{ gas: 100000 });
-                console.log(`set royalties royaltiesProviderArtBlocks ${contract.address} for token ${token}`)
-            }
-            return contract;
-        }
-    );
+    const contract = await deployer.deploy(RoyaltiesProviderArtBlocks, settings.artblocksAddress,  { gas: 1000000 });
```

Comment:
what's settings.artblocksAddress? There is no such constructor arg

Reply:
fixed